File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NUR2016Q00GrdHospListInfo.java
package nta.med.data.model.ihis.nuro;

/**
 * @author DEV-TiepNM
 */
public class NUR2016Q00GrdHospListInfo {
    private String hospCode;
    private String yoyangName;
    private String kanaName;
    private String address;
    private String tel;


    public NUR2016Q00GrdHospListInfo(String hospCode, String yoyangName, String kanaName, String address, String tel) {
        this.hospCode = hospCode;
        this.yoyangName = yoyangName;
        this.kanaName = kanaName;
        this.address = address;
        this.tel = tel;
    }

    public String getHospCode() {
        return hospCode;
    }

    public void setHospCode(String hospCode) {
        this.hospCode = hospCode;
    }

    public String getYoyangName() {
        return yoyangName;
    }

    public void setYoyangName(String yoyangName) {
        this.yoyangName = yoyangName;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getTel() {
        return tel;
    }

    public void setTel(String tel) {
        this.tel = tel;
    }

	public String getKanaName() {
		return kanaName;
	}

	public void setKanaName(String kanaName) {
		this.kanaName = kanaName;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NUR2016Q00GrdHospListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    public class NUR2016Q00GrdHospListInfo
    {
        private String _hospCode;
        private String _yoyangName;
        private String _kanaName;
        private String _address;
        private String _tel;

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String YoyangName
        {
            get { return this._yoyangName; }
            set { this._yoyangName = value; }
        }

        public String KanaName
        {
            get { return this._kanaName; }
            set { this._kanaName = value; }
        }

        public String Address
        {
            get { return this._address; }
            set { this._address = value; }
        }

        public String Tel
        {
            get { return this._tel; }
            set { this._tel = value; }
        }

        public NUR2016Q00GrdHospListInfo() { }

        public NUR2016Q00GrdHospListInfo(String hospCode, String yoyangName, String kanaName, String address, String tel)
        {
            this._hospCode = hospCode;
            this._yoyangName = yoyangName;
            this._kanaName = kanaName;
            this._address = address;
            this._tel = tel;
        }

    }
}